### ISSUE TITLE
Documentation mentions a constructor pattern that doesn't work

### ISSUE THREAD

**Issue by gareth-robinson** [goofballLogic/ld-query#7]
The README.md mentions two constructor patterns

```
var context = LD( { ... etc. } );
var doc = context( data );
```

or

```
var doc = LD( data, { ... context ... } );
```

only the latter actually works. The former gives a 'Cannot convert undefined or null to object' error

**Comment by goofballLogic** [comment]
yup thats a bug

**Comment by gareth-robinson** [comment]
See https://github.com/goofballLogic/ld-query/pull/8

**Comment by goofballLogic** [comment]
Closed by #8
